tanner/tests/test_mysqli.py

```diff
@@ -0,0 +1,144 @@
+import unittest
+import asyncio
+from unittest import mock
+from tanner.utils.asyncmock import AsyncMock
+from tanner.emulators.mysqli import MySQLIEmulator
+
+
+def mock_config(section, value):
```

Comment:
config = {'host':'127.0.0.1' , 'user': 'root', 'passwod':''} and return config[value].
This would be better.

Reply:
Coolz.

Reply:
Do the same in other PR as well.